skbeam/core/roi.py

```diff
@@ -757,3 +759,65 @@ def auto_find_center_rings(avg_img, sigma=1, no_rings=4, min_samples=3,
         edge_pts_xy = edge_pts_xy[~inliers]
 
     return center, image, radii
+
+def find_center_rings_hough(image, num_rings, rad_range, intensity_thresh, sigma, v_range = None):
```

Comment:
This should probably go into the 'feature' module (https://github.com/scikit-beam/scikit-beam/blob/master/skbeam/core/feature.py) as this is about finding and extracting a feature from an image rather than generating a label array for use.  That suggest the function above this should be moved as well and should probably adopt the same return signature as we are proposing here.

Reply:
thanks for the feedback! I made the changes you requested and moved it to feature with a separate plotting function